Testing issues with chalk@2

I've tested cliui against newer dependencies, upgrading chalk to 2.x causes a test failure:
```
  1) cliui
       layoutDSL
         ignores ansi escape codes when measuring padding:

      AssertionError: expected [ Array(9) ] to deeply equal [ Array(5) ]
      + expected - actual

       [
         "  |"
      -  "  "
         "  __|   __|  |   |   _ \\"
      -  "  "
         "  |    |     |   |   __/"
      -  "  "
         " \\__| _|    \\__,_| \\___|"
      -  " "
         "                         "
       ]
```

I've tested the following script:
```js
const chalk = require('chalk');
console.log(JSON.stringify(chalk.blue('line 1\nline 2').split('\n'), null, 4));
```
Result from `chalk@^1.1.2`:
```json
[
    "\u001b[34mline 1",
    "line 2\u001b[39m"
]
```

Result from `chalk@^2.4.2`:
```json
[
    "\u001b[34mline 1\u001b[39m",
    "\u001b[34mline 2\u001b[39m"
]
```

So the big difference is that `chalk@2` adds ansi codes to the start/end of each line.  Upgrading `wrap-ansi` to the latest didn't fix the testing errors, it actually caused more errors though I'm unsure if this is due to an issue with `wrap-ansi` or with `cliui`.

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - "4"
   - "6"
   - "8"
+  - "10"
 after_script: npm run coverage

--- a/package.json
+++ b/package.json
@@ -45,23 +45,23 @@
   "author": "Ben Coe <ben@npmjs.com>",
   "license": "ISC",
   "dependencies": {
-    "string-width": "^2.1.1",
-    "strip-ansi": "^4.0.0",
-    "wrap-ansi": "^2.0.0"
+    "string-width": "^3.1.0",
+    "strip-ansi": "^5.2.0",
+    "wrap-ansi": "^5.1.0"
   },
   "devDependencies": {
-    "chai": "^3.5.0",
-    "chalk": "^1.1.2",
-    "coveralls": "^2.11.8",
-    "mocha": "^3.0.0",
-    "nyc": "^10.0.0",
-    "standard": "^8.0.0",
-    "standard-version": "^3.0.0"
+    "chai": "^4.2.0",
+    "chalk": "^2.4.2",
+    "coveralls": "^3.0.3",
+    "mocha": "^6.0.2",
+    "nyc": "^13.3.0",
+    "standard": "^12.0.1",
+    "standard-version": "^5.0.2"
   },
   "files": [
     "index.js"
   ],
   "engine": {
-    "node": ">=4"
+    "node": ">=6"
   }
 }

